include/quda.h

```diff
@@ -1524,37 +1524,47 @@ extern "C" {
   /**
    * Performs APE smearing on gaugePrecise and stores it in gaugeSmeared
    * @param n_steps Number of steps to apply.
-   * @param alpha  Alpha coefficient for APE smearing.
+   * @param alpha   Alpha coefficient for APE smearing.
```

Comment:
Apologies for being OCD: extra space added here

Reply:
I'll fix all the whitespace stuff with the next round.